console/console-init/ui/src/modules/iot-device/components/ReviewDevice/MetaDataReviewRows.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import { MetaDataReviewRow } from "./MetaDataReviewRow";
+interface IMetaDataReviewRowProps {
+  values: any[];
+  prevkey?: string;
+}
+
+const MetaDataReviewRows: React.FC<IMetaDataReviewRowProps> = ({
```

Comment:
This component is generic and will use in other pages as well. I would suggest generic name MetadataView.